sql/core/benchmarks/CollationBenchmark-jdk21-results.txt

```diff
@@ -1,54 +1,54 @@
+UNICODE_CI                                          44571          44577           8          0.0      445711.8       0.2X
 
-OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1017-azure
+OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - compareFunction:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ---------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY_LCASE                                     6522           6526           4          0.0       65223.9       1.0X
-UNICODE                                              45792          45797           7          0.0      457922.3       0.1X
-UTF8_BINARY                                           7092           7112          29          0.0       70921.7       0.9X
-UNICODE_CI                                           47548          47564          22          0.0      475476.7       0.1X
+UTF8_BINARY_LCASE                                     6823           6836          19          0.0       68230.1       1.0X
+UNICODE                                              45981          45989          12          0.0      459807.8       0.1X
+UTF8_BINARY                                           7421           7422           1          0.0       74212.5       0.9X
+UNICODE_CI                                           48587          48616          41          0.0      485866.3       0.1X
 
-OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1017-azure
+OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - hashFunction:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY_LCASE                                 11716          11716           1          0.0      117157.9       1.0X
-UNICODE                                          180133         180137           5          0.0     1801332.1       0.1X
-UTF8_BINARY                                       10476          10477           1          0.0      104757.4       1.1X
-UNICODE_CI                                       148171         148190          28          0.0     1481705.6       0.1X
+UTF8_BINARY_LCASE                                 11430          11437          10          0.0      114298.7       1.0X
+UNICODE                                          176891         176904          18          0.0     1768912.1       0.1X
+UTF8_BINARY                                        9898           9900           3          0.0       98977.9       1.2X
+UNICODE_CI                                       156862         156869          11          0.0     1568616.3       0.1X
```

Comment:
unrelated

Reply:
Yea, I explained it in the uppermost comment before requesting a review - the JDK version on GHA workers has been bumped, so old resulds are no longer relevant to the "OpenJDK ..." header

Reply:
Discussed this on Slack - since the JDK version and some of the [numbers](https://github.com/apache/spark/pull/46082#pullrequestreview-2008030864) have changed, I will regenerate all the bench results in the separate PR.